woocommerce/woocommerce-android
# Open product screen when onboarding task add product is clicked

Closes: #8628 


### Description
Opens product list screen (which will be empty) when clicking on "Add your first product" task. 

### Testing instructions
- Log into a site that doesn't have any product (create a jurassic ninja site if necessary) 
- Click on "Add your first product" onboarding task
- Check you land empty product list and the bottom sheet to add new product is displayed right away

### Images/gif


https://user-images.githubusercontent.com/2663464/227025211-b553a7af-9c67-42ab-aa31-f2591c090c57.mp4
